models.js

```diff
@@ -72,16 +98,16 @@ debuglog = function () {
             };
             return toFixed(this.value.toString());
         };
+        this.toHashtag = this.toXPath;
```

Comment:
Literals like numbers and strings don't support hashtag transformation. Just act as if it's an xpath.